test/probes/mongodb-core.test.js

```diff
@@ -245,7 +254,16 @@ function makeTests (db_host, host, isReplicaSet) {
         steps.push(exit)
 
         helper.test(emitter, function (done) {
-          db.command(`${dbn}.$cmd`, {create: `coll-${dbn}`}, done)
+          db.command(`${dbn}.$cmd`, {create: `coll-${dbn}`},
+            function (e, data) {
```

Comment:
right now only the collections tests have error logging. Is that the intention? If so, why is it needed here?

Reply:
the only reason is that the collections test were where i was seeing errors to begin with so i wanted to get more information about them. because they're tests i figured there was no harm leaving them in - there is no production cost and getting additional information when tests fail is usually a pretty good thing.